linter/src/test/java/linter/LinterTest.java

```diff
@@ -0,0 +1,20 @@
+package linter;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.io.FileNotFoundException;
+
+
```

Comment:
There's an extra empty line here. It won't cause the apocalypse but try to limit white space where you can (but definitely use white space where you think it's needed!!!!)